utils.sh

```diff
@@ -290,12 +290,14 @@ function generate_metal3_config {
 }
 
 function image_mirror_config {
-    if [ ! -z "${MIRROR_IMAGES}" ]; then
-        . /tmp/mirrored_release_image
-        TAGGED=$(echo $MIRRORED_RELEASE_IMAGE | sed -e 's/release://')
-        RELEASE=$(echo $MIRRORED_RELEASE_IMAGE | grep -o 'registry.ci.openshift.org[^":\@]\+')
+    export ENABLE_LOCAL_REGISTRY=${ENABLE_LOCAL_REGISTRY:-${MIRROR_IMAGES:-}}
```

Comment:
Looks like this one could be removed?

Reply:
The best place is to export ENABLE_LOCAL_REGISTRY common.sh. However, network.sh runs after common.sh and overwrites MIRROR_IMAGES;
https://github.com/openshift-metal3/dev-scripts/blob/67a2d84d7cc22665202359288a85583a2fc9b99f/network.sh#L106

We can export ENABLE_LOCAL_REGISTRY in network.sh or in here.